app/workers/sentry_job.rb

```diff
@@ -0,0 +1,39 @@
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) 2012-2021 the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See docs/COPYRIGHT.rdoc for more details.
+#++
+
+require 'active_job'
+
+class SentryJob < ::ActiveJob::Base
```

Comment:
I recommend inheriting the [`Sentry::SendEventJob`](https://github.com/getsentry/sentry-ruby/blob/master/sentry-rails/app/jobs/sentry/send_event_job.rb) class though. It'll handle some corner cases for you 🙂

Reply:
Hi @st0012 ! Thanks for commenting, will do that

Reply:
np. feel free to open an issue if you need any help on Sentry!

Reply:
I remember why I didn't use it as a base class. Our sentry integration gets loaded dynamically (https://github.com/opf/openproject/pull/9133/files#diff-4d03eef0931ff690167d6bc0a0cfd5327acbf97d3d01da80e24e40c3da7ca5f3R9) which seems to mess with the load path since it won't find /home/oliver/openproject/dev/config/initializers/sentry.rb:17:in `<main>': uninitialized constant Sentry::SendEventJob (NameError) when using that as a base class.

Reply:
ah I see. 

I think the class is only visible after `sentry-rails` is required in the codebase (because the railtie needs to be loaded). and if Sentry is required dynamically, maybe this class should be defined that way too? 
sorry that I don't want to behave like a "sentry expert" and tell you what to do with your code. I just want to help reduce the later maintenance burden with the inheritance. because the long term plan is to make the `SentEventJob` more sophisticated overtime so users won't need to maintain their own jobs anymore.